### PR TITLE
Add Input ~ <input type="button">

### PR DESCRIPTION
**Pull request by nnupoor**
Example added for Issue #818

### CODE REVIEW

File: live-examples/html-examples/input/button.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<fieldset>
+    <legend>Button</legend>
+    <form>
+      <p> These buttons work with access key modifiers too. For Chrome, try `Ctrl + Alt + p`(Mac) / `Alt + p`(Others).</p>
```
Comment (schalkneethling) [inline live-examples/html-examples/input/button.html]: @nnupoor I would recommend we do without these for now. We are going to add JavaScript support to the editor in another round of iteration. You can keep the `accesskey` attribute but, remove this sentence and then the `onclick` JS parts as well.

File: live-examples/html-examples/input/css/button.css [hunk 2]
```diff
@@ -0,0 +1,21 @@
+@font-face {
+    font-family: 'Fira Sans';
+    src: local('FiraSans-Regular'),
+    url('/media/fonts/FiraSans-Regular.woff2') format('woff2');
+}
+
+legend {
+    background-color: #000;
+    color: #ffffff;
```
Comment (schalkneethling) [inline live-examples/html-examples/input/css/button.css]: Nit: Code Style: Prefer short form `#ffffff` => `#fff`

File: live-examples/html-examples/input/css/button.css [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+}
+
+.btn {
+    font: 1rem 'Fira Sans', sans-serif;
+    padding: 1rem;
+    background: #86d1f0;
+    border: none;
+    border-radius: 4px;
+    margin: 4px;
+    outline: none;
```
Comment (schalkneethling) [inline live-examples/html-examples/input/css/button.css]: I would not recommend we add this as it hinders accessibility.

File: live-examples/html-examples/input/button.html [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+      <input class="btn" type="button" value="Click Me" accesskey="p" onclick="alert('I work with Javascript attached to me. But `button` is preferred over me these days. :( ')">
+      <input class="btn" type="button" value="I am disabled" onclick="alert('I won\'t work' )" disabled>
```
Comment (schalkneethling) [inline live-examples/html-examples/input/button.html]: Perhaps add another button showing styling of the different states of a button element? See here for example: https://github.com/mdn/interactive-examples/blob/master/live-examples/html-examples/forms/button.html

**Comment by nnupoor** [comment]
@schalkneethling I'll get these done and raise a PR. Thank you for the suggestions.

**Comment by nnupoor** [comment]
@schalkneethling Fixed the issues. Let me know if I need to do something further. I'll be glad to help. :)

**Pull request merged**